mozilla_django_oidc/auth.py

```diff
@@ -133,8 +125,8 @@ def create_user(self, claims):
             extra_params = {}
 
         return self.UserModel.objects.create_user(
-            email,
-            username=username,
+            username,
```

Comment:
It's funny because until we made a new user model that adhered to Django's UserModel contract, this fork would never be viable to merge into mozilla's project